java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -524,6 +524,52 @@ CompletableFuture<Long> zrangestore(
      */
     CompletableFuture<Object[]> zrankWithScore(String key, String member);
 
+    /**
+     * Returns the rank of <code>member</code> in the sorted set stored at <code>key</code>, where
+     * scores are ordered from the highest to lowest.<br>
+     * To get the rank of <code>member</code> with its score, see {@link #zrevrankWithScore}.
+     *
+     * @see <a href="https://redis.io/commands/zrevrank/">redis.io</a> for more details.
+     * @param key The key of the sorted set.
+     * @param member The member whose rank is to be retrieved.
+     * @return The rank of <code>member</code> in the sorted set, where ranks are ordered from high to
+     *     low based on scores.<br>
+     *     If <code>key</code> doesn't exist, or if <code>member</code> is not present in the set,
+     *     <code>null</code> will be returned.
+     * @example
+     *     <pre>{@code
+     * Long num1 = client.zrevrank("mySortedSet", "member2").get();
+     * assert num1 == 3L; // Indicates that "member2" has the second-highest score in the sorted set "mySortedSet".
```

Comment:
From the redis.io docs, looks like the highest score has rank 0, so second-highest would be rank 1
```suggestion
     * assert num1 == 1L; // Indicates that "member2" has the second-highest score in the sorted set "mySortedSet".
```